hilfe/handbuch/umsaetze/Debitoren/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: layouts/hilfe/single.njk
+title: Debitoren
+weight: 2
+icon: file-invoice
+---
+
+Besonders bei großen Aufträgen und Bestellungen, bevorzugen viele Kunden **auf
+Rechnung zu zahlen**. Diese Option anzubieten kann also zur Kundenzufriedenheit
+beitragen, Sie müssen jedoch auch stets im Auge behahalten, ob
+das Geld schließlich eintrifft. Unter dem Reiter **Debitoren** finden Sie eine Auflistung aller auf
+Rechnung gezahlten Leistungen und können so stets nachvollziehen, wie es um ihre
+Schuldner steht. 
```

Comment:
```suggestion
Besonders bei großen Aufträgen ist eine Bestellung auf Rechnung in vielen Betrieben nicht unüblich.
Unter dem Reiter **Debitoren** finden Sie eine Auflistung aller auf
Rechnung erbrachten Leistungen und können so stets nachvollziehen, wie es um ihre
Schuldner steht. 
```
Finde es nett, dass du die Einleitung auch immer etwas ausschmückst, aber in diesem Fall wirkt das ganze ein bisschen gezwungen und liest sich auch nicht so schön. Glaube, du kannst das noch besser - aber du musst das auch nicht unbedingt für jeden Artikel machen. Also gerne auch etwas kürzer / sachlicher fassen wenn dir grade nichts Gutes einfällt.